### ISSUE TITLE
CUDA Quantum 

### ISSUE THREAD

**Issue by zohimchandani** [SRI-International/QC-App-Oriented-Benchmarks#279]
Can we get CUDA Quantum added to your benchmarks too please?

https://github.com/NVIDIA/cuda-quantum

**Comment by rtvuser1** [comment]
Yes, this is a good idea. However, we should discuss the best approach in a meeting with several QED-C members who work on the benchmarks.  What is the best way to arrange this?

**Comment by rtvuser1** [comment]
There are now several benchmarks that support CUDA-Q: BV, QFT, PE, and HamLib Simulation
